eng/Publishing.props

```diff
@@ -76,6 +76,8 @@
     <!-- Only publish this file from windows x64 so that we don't end up with duplicates -->
     <SdkAssetsToPublish Include="$(ArtifactsShippingPackagesDir)productVersion.txt"
         Condition=" '$(PublishBinariesAndBadge)' == 'true' and '$(OS)' == 'Windows_NT' and '$(Architecture)' == 'x64'" />
+    <SdkAssetsToPublish Include="$(ArtifactsShippingPackagesDir)installer-productVersion.txt"
```

Comment:
I would actually use sdk-productVersion.txt. I think that's more descriptive of the actual purpose.